web/pgadmin/browser/server_groups/servers/pg_timetable/tests/utils.py

```
@@ -0,0 +1,320 @@
+        )
+        old_isolation_level = connection.isolation_level
+        utils.set_isolation_level(connection, 0)
+        pg_cursor = connection.cursor()
+        pg_cursor.execute(
+            """
+            INSERT INTO timetable.chain(
+                chain_name, live
+            ) VALUES (
+                '{0}'::text, true
+            ) RETURNING chain_id;
+            """.format(name)
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)